lib/pbio/drv/usb/usb_ev3.c

```diff
@@ -1153,10 +1174,8 @@ pbio_error_t pbdrv_usb_stdout_tx(const uint8_t *data, uint32_t *size) {
         *size = ptr_len;
     }
     memcpy(ptr, data, *size);
-
-    usb_tx_stdout_is_not_ready = true;
-    pbdrv_cache_prepare_before_dma(ep1_tx_stdout_buf, sizeof(ep1_tx_stdout_buf));
-    usb_setup_tx_dma_desc(CPPI_DESC_TX_STDOUT, ep1_tx_stdout_buf, 2 + *size);
+    ep1_tx_stdout_sz = 2 + *size;
+    pbio_os_request_poll();
```

Comment:
`stdout` is scheduled rather than sent here so we can avoid conflicts with the status message.